ProcessOut/Classes/ProcessOutRequestManager.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  ProcessOutRequest.swift
+//  ProcessOut
+//
+//  Created by Mauro Vime Castillo on 15/2/21.
+//
+
+import Foundation
+
+final class ProcessOutRequestManager {
+    
+    let apiUrl: String
+    let apiVersion: String
+    let defaultUserAgent: String
+  
+    let sessionDelegate: SessionDelegate
+    let retryPolicy: RetryPolicy
+    let urlSession: URLSession
+    
+    init(apiUrl: String, apiVersion: String, defaultUserAgent: String) {
+      self.apiUrl = apiUrl
+      self.apiVersion = apiVersion
+      self.defaultUserAgent = defaultUserAgent
+      
+      let retryPolicy = RetryPolicy()
+      self.retryPolicy = retryPolicy
+      
+      let sessionDelegate = SessionDelegate()
+      sessionDelegate.retrier = retryPolicy
+      self.sessionDelegate = sessionDelegate
+      
+      self.urlSession = URLSession(configuration: .default, delegate: sessionDelegate, delegateQueue: .main)
+    }
+  
+    func HttpRequest(route: String, method: HTTPMethod, parameters: [String: Any]?, completion: @escaping (Data?, ProcessOutException?) -> Void) {
+        guard let projectId = ProcessOut.ProjectId, let authorizationHeader = self.authorizationHeader(user: projectId, password: "") else {
+            completion(nil, ProcessOutException.MissingProjectId)
+            return
+        }
+        
+        do {
+          guard let url = NSURL(string: self.apiUrl + route) else {
```

Comment:
The indentation on this line appears to be inconsistent with the rest of this file. Running the Xcode re-indent tool (`ctrl + i`) on this file also modified a few other lines, such as [111](https://github.com/processout/processout-ios/pull/28/files#diff-b8f36a0e052cfc509e87d0e0165055be4c546d596f39b4537c42b6a79f06dd07R111) and [116](https://github.com/processout/processout-ios/pull/28/files#diff-b8f36a0e052cfc509e87d0e0165055be4c546d596f39b4537c42b6a79f06dd07R116). I recommend running this or a similar formatting tool on your end and committing the result to ensure consistent code formatting.

Reply:
I see now this comment. Sorry, I checked the PR from the Github iOS app a few days ago and didn't see it. Let me re-run this and push it here.